lib/mongo/collection.rb

```diff
@@ -239,6 +244,12 @@ def create(opts = {})
       options = Hash[self.options.reject do |key, value|
         %w(read read_preference read_concern).include?(key.to_s)
       end]
+      # Converting Ruby spelled time series options to server style.
+      [[:time_series,  :timeseries], [:expire_after, :expireAfterSeconds]].each do |(ruby_key, server_key)|
```

Comment:
Can you please use a map for this. A map is still iterable via `map.each do |key, value|`